clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -6530,7 +6530,10 @@ void CodeGenFunction::EmitDeclMetadata() {
 void CodeGenModule::EmitVersionIdentMetadata() {
   llvm::NamedMDNode *IdentMetadata =
     TheModule.getOrInsertNamedMetadata("llvm.ident");
-  std::string Version = getClangFullVersion();
+
+  // FIXME: Choose IncludeRevision based on command-line argument. Including
+  // the Git revision is bad for reproducible builds / caching.
+  std::string Version = getClangFullVersion(/*IncludeRevision=*/false);
```

Comment:
Do we need to upstream this?

Reply:
It's not clear to me we need this, people (including us) should be living on releases, not a clang compiled locally that changes commit-to-commit.